plugins/Toolbox/resources/qml/ToolboxDownloadsGridTile.qml

```diff
@@ -104,8 +104,18 @@ Item
             switch(toolbox.viewCategory)
             {
                 case "material":
-                    toolbox.viewPage = "author"
-                    toolbox.filterModelByProp("packages", "author_id", model.id)
+
+                    // If model has a type, it must be a package
+                    if (model.type)
```

Comment:
I don't like this if-statement to differentiate whether the model is a PackageModel or an AuthorsModel. However after discussing with @ianpaschal , this is fine for now. Probably this will change in the future if we have more models in the Toolbox.